test/e2e/e2e_suite_test.go

```diff
@@ -92,6 +92,8 @@ func init() {
 }
 
 func TestE2E(t *testing.T) {
+	g := NewWithT(t)
```

Comment:
I wonder if this leads to some side effects with other expects somehow.

Looks like we always use the global Expect in other parts of the e2e tests / test framework.

Do you know why it's an issue here but not in other places?

Reply:
Or maybe it's an issue in some other places as well? (e.g. SynchronizedBeforeSuite)

Reply:
I tested it for the other parts. They output the errors greatly because they are inside special ginkgo funcs as you mentioned, e.g. `SynchronizedBeforeSuite`)

The difference to here is, we are here inside a actual test (`func Test<FOO>(t *testing.T){...}`). IMHO it can't lead to side effects because we use the local g variable here.